pkgs/applications/networking/cluster/tilt/default.nix

```diff
@@ -1,24 +1,23 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "tilt";
   /* Do not use "dev" as a version. If you do, Tilt will consider itself
-     running in development environment and try to serve assets from the
-     source tree, which is not there once build completes.  */
-  version = "0.11.3";
+    running in development environment and try to serve assets from the
+    source tree, which is not there once build completes.  */
+  version = "0.16.1";
 
   src = fetchFromGitHub {
-    owner  = "windmilleng";
+    owner  = "tilt-dev";
     repo   = pname;
     rev    = "v${version}";
-    sha256 = "035czgr0rn6gcv24vnlr35n9yvy0fwq4spdzsc76gfxckcbcmzz0";
+    sha256 = "0h3kaq22kg03gi6iffp471ap272lch8x566w7cq25hcdq78hq2v7";
   };
-
-  goPackagePath = "github.com/windmilleng/tilt";
+  vendorSha256 = "1yzqkmjacvzabhw7l70pf3fqkxjd4r1vjniaakwf5k3mr7sj077n";
 
   subPackages = [ "cmd/tilt" ];
 
-  buildFlagsArray = ("-ldflags=-X main.version=${version} -X main.date=2020-01-25");
+  buildFlagsArray = [ "-ldflags=-X main.version=${version} -X main.date=2020-07-23" ];
```

Comment:
What does this date mean? If that's release date, 0.16.1 was released 2020-07-17. If that's build date, it doesn't make sense here and should be undefined.

Reply:
Agreed, I've now removed this. Thanks for the review, a new version has come out since this - lets continue in #94827. Cheers